frontend_app_admin_workspace_user/src/component/AddUserForm.jsx

```diff
@@ -113,20 +113,30 @@ export class AddUserForm extends React.Component {
                 key={p.id}
                 data-cy={`profile__list__item__${p.slug}`}
               >
-                <input
-                  type='radio'
-                  name='newUserProfile'
-                  id={p.slug}
-                  value={p.slug}
-                  checked={state.newUserProfile === p.slug}
-                  onChange={this.handleChangeNewUserProfile}
-                />
-
-                <div className='d-flex align-items-center'>
+                <div className='d-flex align-items'>
+                  <div className='userrole__role__input'>
+                    <input
+                      type='radio'
+                      name='newUserProfile'
+                      id={p.slug}
+                      value={p.slug}
+                      checked={state.newUserProfile === p.slug}
+                      onChange={this.handleChangeNewUserProfile}
+                    />
+                  </div>
+
                   <div className='userrole__role__icon mx-2' style={{ color: p.hexcolor }}>
                     <i className={`fa fa-fw fa-${p.faIcon}`} />
                   </div>
-                  {props.t(p.label) /* this trad key is declared in frontend/helper.js, object PROFILE */}
+
+                  <div className='profile__list__item__content'>
```

Comment:
If you add "__content" (which is good), you need to add it the children of that div.
So "profile__list__item__label" should become "profile__list__item__content__label"
Update stylus accordingly